pkg/registry/apis/dashboard/legacy/rv.go

```diff
@@ -0,0 +1,45 @@
+package legacy
+
+import (
+	"context"
+	"time"
+
+	"github.com/grafana/grafana/pkg/infra/db"
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+type ResourceVersionLookup = func(ctx context.Context) (int64, error)
+
+func GetResourceVersionLookup(sql db.DB, table string) ResourceVersionLookup {
```

Comment:
If we wan't to reuse this for more resource maybe there is a better package than /dashboards/legacy. WDYT?

Reply:
moved